### PR TITLE
Add doc for data model

### LINKED ISSUE #89: Add doc for data model
Add doc for data model

Effort: 1 man day

### PR DESCRIPTION
**Pull request by crbrox**
Close #89

@fgalan 
@dmoranj 
@jcalderin

### CODE REVIEW

File: documentation/models.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+by the Portal for giving it back when the Portal requests it
+
+Fields:
+* **_id** *ObjectId*: unique object id used by mongoDB
+* **name**  *string*: name of the rule
+* **service** *string* : service which the rule belongs to.
+* **subservice** *string*: subservice which the rule belongs to.
+* **tex** *string*: EPL sentence for the rule, to be propagated to core
+* **action** *object*: action to be executed when the rule is fired. Each action type has different field set as described in [Plain rules](plain_rules.md#actions)
+    * **type** ( *string* ): type of action.
```
Comment (fgalan) [inline documentation/models.md]: What about `template` and `parameters`? The example below shows they are also subfields inside `action`, but they are not mentioned here.

File: documentation/models.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+```
+
+
+### Executions
+The collection 'executions' stores information about the sucessfully executed actions. A TTL index deletes exectutions older than ine day
+
+Fields:
+* **_id** *ObjectId*: unique object id used by mongoDB
+* **id**  *string*: entity id that fired the rule
+* **service** *string* : service which the fired rule belongs to.
```
Comment (fgalan) [inline documentation/models.md]: Looking to the example below, it seems that `name` description is missing.

File: documentation/models.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+	"lastTime" : ISODate("2015-04-16T14:05:13.922Z")
+}
+```
+
+
+
+
+<a name="indexes"></a>
+## Indexes
+
```
Comment (fgalan) [inline documentation/models.md]: As helper, the ensureIndex() command to run to create each index could be included.

File: documentation/models.md (hunk 2 above)
Comment (crbrox) [inline documentation/models.md]: Fixed in 62cc5e9

File: documentation/models.md (hunk 3 above)
Comment (crbrox) [inline documentation/models.md]: It is created by perseo-fe, not manual creation is needed, but we can add the command if you think it is worth

File: documentation/models.md (hunk 1 above)
Comment (crbrox) [inline documentation/models.md]: They are documented in [Plain rules](plain_rules.md#actions) as pointed out in doc. Not all actions have all the fields, but we can add a brief reference of those fields if you want

File: documentation/models.md (hunk 3 above)
Comment (fgalan) [inline documentation/models.md]: I think it helps for people with knownledgue in MongoDB.

File: documentation/models.md (hunk 1 above)
Comment (fgalan) [inline documentation/models.md]: In that case, maybe we could add a "wildcard" bullet, like this one, just after `** type`:

```
** Other subfields, depending on the rule type.
```

File: documentation/models.md (hunk 3 above)
Comment (crbrox) [inline documentation/models.md]: Fixed 9c684c3

File: documentation/models.md (hunk 1 above)
Comment (crbrox) [inline documentation/models.md]: Fixed 7e1ec97

**Comment by fgalan** [comment]
LGTM

**Pull request merged**